en:
  errors:
    auth:
      unauthorized_action: "Unauthorized action"